comment/admin.py

```diff
@@ -1,5 +1,6 @@
 from django.contrib import admin
-from comment.models import Comment
+
+from comment.models.comments import Comment
```

Comment:
Since we imported the models in `__init__` we can terminate the module name from the import statement.
`from comment.models import Comment`  <-- I prefer this form.